spring-cloud-stream-binder-kafka-docs/src/main/asciidoc/overview.adoc

```diff
@@ -429,4 +430,69 @@ On the other hand, if auto topic creation is disabled on the server, then care m
 
 If you want to have full control over how partitions are allocated, then leave the default settings as they are, i.e. do not exclude the kafka broker jar and ensure that `spring.cloud.stream.kafka.binder.autoCreateTopics` is set to `true`, which is the default.
 
+== Kafka Stream Binding Capabilities of Spring Cloud Stream
 
+Spring Cloud Stream Kafka support also includes a binder specifically designed for Kafka Streams binding.
+Using this binder, applications can be written that leverage the Kafka Streams API.
+For more information on Kafka Streams, see https://kafka.apache.org/0110/documentation/streams/developer-guide[Kafka Streams API Developer Manual]
```

Comment:
Looks like we can have versionless link: https://kafka.apache.org/documentation/streams/developer-guide